flexget/components/sites/sites/piratebay.py

```diff
@@ -163,7 +163,11 @@ def search(self, task, entry, config=None):
                 if href.startswith('/'):  # relative link?
                     href = self.url + href
                 entry['url'] = href
-                tds = link.parent.parent.parent.find_all('td')
+                row = link.parent.parent.parent
+                if row.find_all('a', attrs={'class': 'detDesc'})[0].contents[0] == "piratebay ":
```

Comment:
if no `class: detDesc` is found this will crash, more robust approach would be preferable

Reply:
Ok, I tried to make it more robust